Debugger attached.

> testing123-website@1.0.0 start
> node index.html

Debugger attached.
Waiting for the debugger to disconnect...
/workspaces/testing123/index.html:1
<html>
^

SyntaxError: Unexpected token '<'
    at wrapSafe (node:internal/modules/cjs/loader:1662:18)
    at Module._compile (node:internal/modules/cjs/loader:1704:20)
    at Object..js (node:internal/modules/cjs/loader:1895:10)
    at Module.load (node:internal/modules/cjs/loader:1465:32)
    at Function._load (node:internal/modules/cjs/loader:1282:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:171:5)
    at node:internal/main/run_main_module:36:49

Node.js v22.17.0
Waiting for the debugger to disconnect...
Debugger attached.

> testing123-website@1.0.0 start
> node index.html

Debugger attached.
Waiting for the debugger to disconnect...
/workspaces/testing123/index.html:1
<html>
^

SyntaxError: Unexpected token '<'
    at wrapSafe (node:internal/modules/cjs/loader:1662:18)
    at Module._compile (node:internal/modules/cjs/loader:1704:20)
    at Object..js (node:internal/modules/cjs/loader:1895:10)
    at Module.load (node:internal/modules/cjs/loader:1465:32)
    at Function._load (node:internal/modules/cjs/loader:1282:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:171:5)
    at node:internal/main/run_main_module:36:49

Node.js v22.17.0
Waiting for the debugger to disconnect...
Debugger attached.

> testing123-website@1.0.0 start
> node index.html

Debugger attached.
Waiting for the debugger to disconnect...
/workspaces/testing123/index.html:1
<html>
^

SyntaxError: Unexpected token '<'
    at wrapSafe (node:internal/modules/cjs/loader:1662:18)
    at Module._compile (node:internal/modules/cjs/loader:1704:20)
    at Object..js (node:internal/modules/cjs/loader:1895:10)
    at Module.load (node:internal/modules/cjs/loader:1465:32)
    at Function._load (node:internal/modules/cjs/loader:1282:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:171:5)
    at node:internal/main/run_main_module:36:49

Node.js v22.17.0
Waiting for the debugger to disconnect...
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
Hello from index.js!
Waiting for the debugger to disconnect...
Waiting for the debugger to disconnect...
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
just making sure its running!
Server listening at http://localhost:3000
Error: ENOENT: no such file or directory, stat '/workspaces/testing123/public/index.html'
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
just making sure its running!
Server listening at http://localhost:3000
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
just making sure its running!
Server listening at http://localhost:3000
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
just making sure its running!
Server listening at http://localhost:3000
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
just making sure its running!
Server listening at http://localhost:3000
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
Server listening at http://localhost:3000
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
Server listening at http://localhost:3000
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
Server listening at http://localhost:3000
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
Waiting for the debugger to disconnect...
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1940:16)
    at listenInCluster (node:net:1997:12)
    at Server.listen (node:net:2102:7)
    at Function.listen (/workspaces/testing123/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/workspaces/testing123/index.js:12:9)
    at Module._compile (node:internal/modules/cjs/loader:1730:14)
    at Object..js (node:internal/modules/cjs/loader:1895:10)
    at Module.load (node:internal/modules/cjs/loader:1465:32)
    at Function._load (node:internal/modules/cjs/loader:1282:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1976:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -98,
  syscall: 'listen',
  address: '::',
  port: 3000
}

Node.js v22.17.0
Waiting for the debugger to disconnect...
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
Waiting for the debugger to disconnect...
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1940:16)
    at listenInCluster (node:net:1997:12)
    at Server.listen (node:net:2102:7)
    at Function.listen (/workspaces/testing123/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/workspaces/testing123/index.js:12:9)
    at Module._compile (node:internal/modules/cjs/loader:1730:14)
    at Object..js (node:internal/modules/cjs/loader:1895:10)
    at Module.load (node:internal/modules/cjs/loader:1465:32)
    at Function._load (node:internal/modules/cjs/loader:1282:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1976:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -98,
  syscall: 'listen',
  address: '::',
  port: 3000
}

Node.js v22.17.0
Waiting for the debugger to disconnect...
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
Waiting for the debugger to disconnect...
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1940:16)
    at listenInCluster (node:net:1997:12)
    at Server.listen (node:net:2102:7)
    at Function.listen (/workspaces/testing123/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/workspaces/testing123/index.js:12:9)
    at Module._compile (node:internal/modules/cjs/loader:1730:14)
    at Object..js (node:internal/modules/cjs/loader:1895:10)
    at Module.load (node:internal/modules/cjs/loader:1465:32)
    at Function._load (node:internal/modules/cjs/loader:1282:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1976:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -98,
  syscall: 'listen',
  address: '::',
  port: 3000
}

Node.js v22.17.0
Waiting for the debugger to disconnect...
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
Waiting for the debugger to disconnect...
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1940:16)
    at listenInCluster (node:net:1997:12)
    at Server.listen (node:net:2102:7)
    at Function.listen (/workspaces/testing123/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/workspaces/testing123/index.js:12:9)
    at Module._compile (node:internal/modules/cjs/loader:1730:14)
    at Object..js (node:internal/modules/cjs/loader:1895:10)
    at Module.load (node:internal/modules/cjs/loader:1465:32)
    at Function._load (node:internal/modules/cjs/loader:1282:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1976:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -98,
  syscall: 'listen',
  address: '::',
  port: 3000
}

Node.js v22.17.0
Waiting for the debugger to disconnect...
Debugger attached.

> testing123-website@1.0.0 start
> node index.js

Debugger attached.
Waiting for the debugger to disconnect...
node:events:496
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1940:16)
    at listenInCluster (node:net:1997:12)
    at Server.listen (node:net:2102:7)
    at Function.listen (/workspaces/testing123/node_modules/express/lib/application.js:635:24)
    at Object.<anonymous> (/workspaces/testing123/index.js:12:9)
    at Module._compile (node:internal/modules/cjs/loader:1730:14)
    at Object..js (node:internal/modules/cjs/loader:1895:10)
    at Module.load (node:internal/modules/cjs/loader:1465:32)
    at Function._load (node:internal/modules/cjs/loader:1282:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1976:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -98,
  syscall: 'listen',
  address: '::',
  port: 3000
}

Node.js v22.17.0
Waiting for the debugger to disconnect...
